### ISSUE TITLE
Infinite view did not works well with UICollectionView on iOS6  (or PSTCollectionView on iOS < 6)

### ISSUE THREAD

**Issue by thomasdao** [samvermette/SVPullToRefresh#71]
Hi,

Thanks for awesome library :+1: .  

The pull to refresh view work well for UICollectionView, since UICollectionView is subclass of UIScrollView. It seems the infinite view doesn't show on UICollectionView though. 

I believe you can reproduce by checkout the sample about UICollectionView or PSTCollectionView ( https://github.com/steipete/PSTCollectionView) then add SVPullToRefresh. I can see the infinite scroll handler is triggered, however the view is not there.

**Comment by samvermette** [comment]
Have you tried setting the activity indicator's style to white? http://cl.ly/KY4j

**Comment by thomasdao** [comment]
That's nice, I set the indicator to white like below 

```
-(void)viewWillAppear:(BOOL)animated {
    [self.collectionView.infiniteScrollingView setActivityIndicatorViewStyle:UIActivityIndicatorViewStyleWhite];
    [self.collectionView addInfiniteScrollingWithActionHandler:^{
        NSLog(@"Infinitescroll");
    }];
    [super viewWillAppear:animated];
}
```

But I still can't see the indicator. Probably I use it wrong way, how do you make it work?

**Comment by samvermette** [comment]
Try setting the activityIndicator style _after_ calling addInfiniteScrollingWithActionHandler:

**Comment by hemangshah** [comment]
Have you ever solved this? I'd the similar issue with iOS 7.0 and above.
@samvermette , just awesome control thanks a ton! :+1:
